src/adapter/receiver.rs

```diff
@@ -1,10 +1,11 @@
 use rustdoc_types::{GenericArgs, Type};
+use std::borrow::Cow;
```

Comment:
nit (and I'll eventually figure out how to get rustfmt to do this) but for readability it's usually a good idea to have built-in imports (`std / core` etc.) at the top then 3rd party imports afterward.

Reply:
I see, good point! I could save without format in this case!